### ISSUE TITLE
Storage adapter absence in config leads to strange error

### ISSUE THREAD

**Issue by timbset** [reimagined/resolve#614]
I do something like this and start server (or start building) without specifying of this env variable.
```
{
  "storage": {
    "adapter": "${RESOLVE_STORAGE_ADAPTER}"
  }
}
```
This leads to the following error:
```
ERROR in ./node_modules/resolve-scripts/dist/runtime/server/event_store.js
Module not found: Error: Can't resolve 'D:\Repositories\okrserver' in 'D:\Repositories\okrserver\node_modules\resolve-scripts\dist\runtime\server'
 @ ./node_modules/resolve-scripts/dist/runtime/server/event_store.js 21:21-54
 @ ./node_modules/resolve-scripts/dist/runtime/server/socket_handler.js
 @ ./node_modules/resolve-scripts/dist/runtime/server/index.js
 @ multi babel-regenerator-runtime ./node_modules/resolve-scripts/dist/runtime/server/index.js
```
I expect to see an error message explaining to me what I am doing wrong.

**Comment by TatyanaRyzh** [comment]
Hi timbset,
 
We've reproduced your scenario and will fix it as soon as possible.
 
Thank you for helping us improve reSolve!

**Comment by TatyanaRyzh** [comment]
Fixed in [V0.12.1](https://github.com/reimagined/resolve/releases/tag/V0.12.1)
